File: kattis/BatterUp.java
package kattis;

import java.util.Scanner;

public class BatterUp {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int NoAB = scanner.nextInt();
        int timesAtBat = NoAB;

        double sum = 0;
        for (int i=0; i < NoAB; i++) {
            int action = scanner.nextInt();

            if (action == -1) timesAtBat--;
            else sum += action;
        }
        System.out.println(sum / timesAtBat);
    }
}

File: kattis/easy/ccpp/batterup.cpp
 //
// Created by BBJ on 03.01.2024.
//
#include <iostream>
using std::cin;
using std::cout;

int main() {
    int noAB; cin >> noAB;
    int timesAtBat = noAB;

    long double sum = 0;
    for (int i=0; i < noAB; ++i) {
        int action; cin >> action;

        if (action == -1) timesAtBat--;
        else sum += action;
    }

    cout << (sum / timesAtBat);

    return 0;
}